.travis.yml

```diff
@@ -2,7 +2,17 @@ language: generic
 
 script: bash ./deploy.sh
 
+branches:
+  only:
+    - gh-pages
+
 env:
   global:
   - ENCRYPTION_LABEL: 104fbe69e8fa
   - COMMIT_AUTHOR_EMAIL: travis-ci@w3.org
+  - URL="https://w3c.github.io/permissions/"
+  - DECISION="https://lists.w3.org/Archives/Public/public-webappsec/2015Mar/0170.html"
+  - secure: SneznWMBm52ct+mFI6NAVrB8k/BjO/H/hjR1eB3tjPwF0vzpL5fnbD1mmlNmVJMTWMsSUfI917JotA+7lfwoWJuHwlq2WfzQ0ybfvtuqjo7LS22T5X8IFYmCyVNR5asbJha/LNhqUyGDsZlPBgbKK0hqTd8eWjjQTztnlSrorGM=
```

Comment:
Can you `# comment` what secrets are in this string? I assume it's the token from https://github.com/w3c/echidna/wiki/How-to-use-Echidna-with-ReSpec-and-GitHub#the-token?